src/snyk/extension.ts

```diff
@@ -69,6 +70,7 @@ import { DailyScanJob } from './snykOss/watchers/dailyScanJob';
 class SnykExtension extends SnykLib implements IExtension {
   public async activate(vscodeContext: vscode.ExtensionContext): Promise<void> {
     extensionContext.setContext(vscodeContext);
+    SecretStorageAdapter.init(vscodeContext);
```

Comment:
Maybe move this to `initializeExtension` function, since all initialisation happens there.